assets/css/materialize.css

```diff
@@ -10741,3 +10741,25 @@ div.slack {
     font-size: 8px;
   }
 }
+
+/*Community Meetings Table */
+table {
```

Comment:
Redefining these styles in this way will have a site-wide affect. Namespace these styles or use selectors.

Reply:
Should not change global style.